### PR TITLE
RET-3549: Aligning multiple file uploads for applications and notifications

### PR DESCRIPTION
**Pull request by JackReevies**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RET-3549

### Change description ###

Aligning multiple file uploads for applications and notifications

**Does this PR introduce a breaking change?** (check one with "x")

```
[] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/MarkdownHelper.java [hunk 1]
```diff
@@ -23,7 +23,16 @@ public static String createTwoColumnTable(String[] header, List<String[]> rows)
             throw new IllegalArgumentException("header array should contain two items");
```
Comment (paul-peasegood) [inline src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/MarkdownHelper.java:23]: What is header is greater than 2?

Comment (JackReevies) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/MarkdownHelper.java:23]: To make a two column table, the String array passed in must contain two items

**Review by paul-peasegood — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/utils/DocumentFixtures.java [hunk 2]
```diff
@@ -34,4 +34,5 @@ public static DocumentType getDocumentType() {
     public static DocumentType getDocumentType(String documentName) {
         return DocumentType.from(getUploadedDocumentType(documentName));
     }
+
```
Comment (Itay-Kirsh) [inline src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/utils/DocumentFixtures.java:37]: Why?

File: src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/TseViewApplicationsController.java [hunk 3]
```diff
@@ -36,10 +34,10 @@
 public class TseViewApplicationsController {
 
     private final VerifyTokenService verifyTokenService;
-    private final TornadoService tornadoService;
-    private final UserService userService;
+    private final TseService tseService;
     private static final String INVALID_TOKEN = "Invalid Token {}";
 
+
```
[the remark this answers is not in the document]
Comment (JackReevies) [reply src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/controllers/TseViewApplicationsController.java]: yeah me neither, dunno why this made it in - gone

**Review by Itay-Kirsh — APPROVED** [review]

**Pull request merged**